tests/test_sealed_box.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2013 Donald Stufft and individual contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+import binascii
+
+import pytest
+
+from nacl.encoding import HexEncoder
+from nacl.public import PrivateKey, PublicKey, SealedBox
+
+
+VECTORS = [
+    # privalice, pubalice, plaintext, encrypted
+    (
+        b"77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a",
```

Comment:
I've put at https://gist.github.com/lmctv/4a6de03c614985da68fee462c993751f a C driver for libsodium seal/seal_open APIs.

After compiling with a command line like 
```
cc sealbox_test.c -o sealbox_test \
                   -Isrc/libsodium/src/libsodium/include/ \
                   -L./build/temp.linux-x86_64-3.5/lib/ -lsodium
```
You can use it to generate test lines on standard output by executing
```
./sealbox_test  N [minmsglen[maxmsglen]
```
and to test existing lines in the format <sk>,<pk>,<msg>,<encrypted> from standard input by executing
```
./sealbox_test
```
A pure libsodium round-trip test can be run like
```
 ./sealbox_test  4 16 64| ./sealbox_test
```
In this case, 4 test lines will get generated and tested, with message lengths between 16 and 64 bytes.